docs/src/main/docs/guides/07_datasource.adoc

```diff
@@ -0,0 +1,364 @@
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Setting Up Data Sources
+:description: Helidon MP Data Source Guide
+:keywords: helidon, guide, datasource, microprofile
+
+This guide shows how to configure and use named
+https://docs.oracle.com/javase/8/docs/api/javax/sql/DataSource.html[`DataSource`]s
+in your Helidon MP application.
+
+== What You Need
+
+|===
+|About 20 minutes
+|<<about/03_prerequisites.adoc,Helidon Prerequisites>>
+|The Maven coordinates for your database vendor's JDBC-compliant database driver library
+|===
+
+== Add Your Database Driver to Your Helidon MP Application's Runtime Classpath
+
+Make sure your database vendor's driver library is present on your
+application's runtime classpath.  In a Maven project, add an
+appropriate
+https://maven.apache.org/ref/3.6.1/maven-model/maven.html#class_dependency[`<dependency>`]
+element as a child element of the `<dependencies>` element in your
+`pom.xml`, referencing the driver library.  Here's an example that
+includes the
+https://search.maven.org/classic/#artifactdetails%7Ccom.h2database%7Ch2%7C1.4.199%7Cjar[H2
+database driver library]:
+
+[source,xml]
+----
+<dependency>
+    <groupId>com.h2database</groupId>
+    <artifactId>h2</artifactId>
+    <version>1.4.199</version>
+    <scope>runtime</scope> <!--1-->
+</dependency>
+----
+<1> Note the `<scope>` is `runtime`.
+
+== Add a Helidon Connection Pool Extension to Your Helidon MP Application's Runtime Classpath
+
+Helidon contains two <<extensions/01_overview.adoc,extensions>> that
+supply _connection pools_, infrastructure that allows efficient use of
+database connections:
+
+1. <<extensions/02_cdi_datasource-hikaricp.adoc,HikariCP extension>>:
+   an extension that integrates the
+   https://github.com/brettwooldridge/HikariCP[Hikari connection pool]
+   into Helidon MP applications
+
+2. <<extensions/02_cdi_datasource-ucp.adoc,Oracle Universal Connection
+   Pool extension>>: an extension that integrates
+   https://docs.oracle.com/en/database/oracle/oracle-database/19/jjucp/index.html[Oracle's
+   Universal Connection Pool] into Helidon MP applications
+
+Each connection pool offers tradeoffs in terms of size, efficiency and
+features.
+
+=== Add the Hikari Connection Pool to the Runtime Classpath
+
+If you choose to integrate the Hikari connection pool into your
+Helidon MP application, then add an appropriate
+https://maven.apache.org/ref/3.6.1/maven-model/maven.html#class_dependency[`<dependency>`]
+element as a child element of the `<dependencies>` element in your
+`pom.xml`, referencing the HikariCP extension:
+
+[source,xml,subs="attributes+"]
+----
+<dependency>
+    <groupId>io.helidon.integrations.cdi</groupId>
+    <artifactId>helidon-integrations-cdi-datasource-hikaricp</artifactId>
+    <scope>runtime</scope> <!--1-->
+</dependency>
+----
+<1> Note the `<scope>` is `runtime`.
```

Comment:
How about couple more words about the to explain why the runtime scope ?
This is I guess common to all CDI extensions we provide so maybe we could document that in a common place or say something like "Note the `<scope>` is `runtime` since it is not needed during compilation"

Reply:
Fixed; I just removed the note